sympy/core/tests/test_symbol.py

```diff
@@ -391,3 +395,33 @@ def test_disambiguate():
     assert disambiguate(*t7) == (y*y_1, y_1)
     assert disambiguate(Dummy('x_1'), Dummy('x_1')
         ) == (x_1, Symbol('x_1_1'))
+
+
+def test_issue_gh_16734():
+    try:
+        import pyodide_js  # noqa
+    except ImportError:
+        pass
+    else:
+        skip('Cannot create threads under pyodide')
```

Comment:
Are there other environments where threading can fail? Would it be better to just skip if `thread.start()` raises RuntimeError?

Reply:
I don't know of any other context where it would fail. Probably best would be to have a `skip_if_running_in_pyodide()` helper. There is one other place that would use it.

Reply:
I'm not sure where to put that though....